.github/workflows/lint.yml

```diff
@@ -29,13 +29,13 @@ jobs:
       with:
         go-version: ${{ env.DEFAULT_GO_VERSION }}
     - uses: actions/checkout@v2
-    - uses: actions/cache@v2.1.4
```

Comment:
@MrAlias @owais Can't we simply use `actions/cache@v2`?

Reply:
I've no idea how it works :) as long as it doesn't break the CI, I really don't care :D

Reply:
SGTM